app/controllers/store_controller.rb

```diff
@@ -0,0 +1,7 @@
+class StoreController < ApplicationController
+  def index
+    @restaurants =Restaurant.order(:name )
+    @reviews = Review.order(:restaurant_id)
```

Comment:
What could have been better was to get all the restaurants from the db, and each restaurant has their own reviews right? something like `restaurant.reviews` would have given you all the reviews of that restaurant.
This can easily be done with rails.  [Read this](http://blog.arkency.com/2013/12/rails4-preloading/) to understand how.